_docs/overview/multi-signature-security.md

```diff
@@ -74,8 +74,10 @@ is the co-owner of the safe deposit box. A successor trustee could then access t
 
 ### Option 2: Distributed custody of keys
 
-Another option is to distribute a number of keys to trusted individuals, or signatories,
-which has the following advantages:
+A 2-of-5 withdrawal policy is recommended, with private keys distributed to a number
+of trusted individuals, or signatories. The extra key, five rather than the recommended
+four in Option 1, mitigates the extra risk of signatories not sufficiently protecting
+the key against loss. This has the following advantages:
```

Comment:
Is the commit comment mixed up about loss vs theft? The git text says you removed "loss" but it looks like you removed "theft".

Reply:
Yes, it should have read "loss". This has since been reinstated (after issue #18 discussion) in a new commit.